pages/gall/[id]/index.tsx

```diff
@@ -179,9 +223,10 @@ const Gall = ({ species, imagePaths }: Props): JSX.Element => {
                         <Row>
                             <Col>
                                 <strong>Further Information:</strong>
+                                <br/>
+                                <Button variant="outline-secondary" onClick={sortSourceList}>{sourceList.sort.text}</Button>
```

Comment:
I think we should split these into 2 controls, one for the sort field and one for the sort order.

From a layout standpoint push the sort controls over to the right and remove the <br />.